src/components/IntlSelectForm.js

```diff
@@ -3,14 +3,17 @@
 		components to make them more reusable.  This now gets passed into Selection form
 		to render the proper content for the select form elements.
 */
-
+/*
+	I have added the delay and animation styles to this component, which are passed in
+	via props from the CryptoTrack component.
+ */
 import React from 'react';
 import Radium from 'radium';
 import RadiumVars from '../RadiumVariables';
 
 const IntlSelectForm = props => {
 	return (
-				<select style={styles.selectForm__form}>
+				<select style={[styles.selectForm__form, props.delay, props.anim]}>
```

Comment:
You can add any delay and keyframe combination you want to this `select` element now.  I am using it to draw attention to the element to show that the user needs to take an action on this element.